ServiceRequest/ServiceRequest-NonWGSTestOrderForm-FetalScenario-Example.json

```diff
@@ -0,0 +1,117 @@
+{
+	"resourceType": "ServiceRequest",
+	"id": "ServiceRequest-NonWGSTestOrderForm-FetalScenario-Example",
+	"extension": [
+		{
+			"url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-Coverage",
+			"valueCodeableConcept": {
+				"coding": [
+					{
+						"system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-FundingCategory",
+						"code": "nhs",
+						"display": "NHS"
+					}
+				]
+			}
+		}
+	],
+	"status": "active",
+	"intent": "order",
+	"category": [
+		{
+			"coding": [
+				{
+					"system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-GenomeSequencingCategory",
+					"code": "rare-disease-non-wgs",
+					"display": "Rare Disease - Non-WGS"
+				}
+			]
+		}
+	],
+	"priority": "urgent",
+	"code": {
+		"coding": [
+			{
+				"system": "https://fhir.nhs.uk/CodeSystem/England-GenomicTestDirectory",
+				"code": "R21",
+				"display": "Fetal anomalies with a likely genetic cause"
+			}
+		]
+	},
+	"subject": {
+		"reference": "Patient/Patient-FoetusOfRyanneBoulder-Example",
+		"identifier": [
+			{
+				"system": "urn:oid:2.16.840.1.113883.2.1.3.2.4.18.24",
+				"value": "FT-RWT13521",
+				"assigner": {
+					"identifier": {
+						"system": "https://fhir.nhs.uk/Id/ods-organization-code",
+						"value": "RAX"
+					}
+				}
+			}
+		]
+	},
+	"authoredOn": "2023-09-25",
+	"requester": {
+		"reference": "PractitionerRole/PractitionerRole-LoisLaneKingstonClinicalGenetics-Example"
+	},
+	"reasonCode": [
+		{
+			"coding": [
+				{
+					"system": "http://snomed.info/sct",
+					"code": "243787009",
+					"display": "Antenatal screening"
+				}
+			]
+		}
+	],
+	"supportingInfo": [
+		{
+			"reference": "Observation/Observation-RiskOfInfection-Example"
```

Comment:
Remove as this is recorded against specimen (using the specialHandling extension of risk of infection is true)